src/components/application_manager/include/application_manager/service.h

```diff
@@ -218,7 +226,8 @@ class Service {
    * @param message message for validation
    * @return true if message is valid according to schema, otherwise false
    */
-  virtual bool ValidateMessageBySchema(const Message& message) = 0;
+  virtual MessageValidationResult ValidateMessageBySchema(
+      const Message& message) = 0;
```

Comment:
@AKalinich-Luxoft different use cases requires different behavior. Sometimes we need not only bool